src/libecalc/presentation/exporter/configs/configs.py

```diff
@@ -642,15 +640,15 @@ def aggregator(frequency: Frequency) -> Aggregator:
                     name="fromShoreConsumption",
                     title="Total Electricity Consumed From Power-From-Shore",
                     unit=Unit.GIGA_WATT_HOURS,
-                    query=ElectricityGeneratedQuery(
+                    query=PowerConsumptionQuery(
```

Comment:
This should have been consumption. Since default `power` from generator set does not calculate `cable_loss` the generated power == consumed power.

Reply:
good point, I think it is worth adding that comment to the query itself, to avoid misunderstanding or change in name later etc

Reply:
or, problably best to add here